### ISSUE TITLE
Fast paas : getting fast paas issues in tag picker when item Limit is selected. i.e Ensures every ARIA input field has an accessible name and Ensures elements with an ARIA role that require child roles contain them.

### ISSUE THREAD

**Issue by Sandhya-aluri** [microsoft/fluentui#15829]
### Environment Information

- **Package version(s)**:  "@fluentui/react": "^7.123.4"
- **Browser and OS versions**: Chrome and Windows 10

### Describe the issue:
In tag picker control , i have delcared item limit as 3. When 3 items are selected , fast paas is failing with below 2 issues. 
1. Ensures elements with an ARIA role that require child roles contain them.
2. Ensures every ARIA input field has an accessible name.

#### Actual behavior:
In tag picker control ,when item limit is selected. fast paas is failing with below 2 issues. 
1. Ensures elements with an ARIA role that require child roles contain them.
2. Ensures every ARIA input field has an accessible name.

#### Expected behavior:
In tag picker control ,when item limit is selected. fast paas should pass.
<img width="628" alt="tagpicker1" src="https://user-images.githubusercontent.com/48669559/98091495-84baaf00-1eab-11eb-9e6b-8131d2791a57.PNG">
<img width="960" alt="tagpicker2" src="https://user-images.githubusercontent.com/48669559/98091506-87b59f80-1eab-11eb-947d-499888ddcec8.png">

**Comment by paulgildea** [comment]
@Sandhya-aluri Thanks for filing this issue.

@smhigley FYI on the model.

@joschect on the current design.
